src/serialize.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+
+"use strict";
+
+import utils = require("./utils");
+import kernel = require("./kernel");
+
+import IKernelMsg = kernel.IKernelMsg;
+
+
+function _deserializeArrayBuffer(buf: ArrayBuffer): IKernelMsg {
```

Comment:
If it's not exported, it can't be accessed outside the module. No need for a leading underscore. Prefer placing exported functions at the top of the module.